### PR TITLE
Added paragraph to explain what's included in a Self-Description

### PR DESCRIPTION
**Pull request by tom-rm-meyer-ISST**
- Self-Description of the component
- Self-Description of the offered endpoints
- Self-Description of the offered data

### CODE REVIEW

**Comment by tom-rm-meyer-ISST** [comment]
Proposal to solve https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/issues/213

File: documentation/3_Layers_of_the_Reference_Architecture_Model/3_5_System_Layer/3_5_4_Metadata_Broker.md [hunk 1]
```diff
@@ -1,12 +1,17 @@
 ### Metadata Broker ###
 
-The IDS Metadata Broker consists of an IDS Connector (see Section [3.5.2](./3_5_2_IDS_Connector.md#ids-connector)), an endpoint for the registration, publication, maintenance, and query of Self-Descriptions. Therefore, for any interaction with the IDS Metadata Broker, the processes defined on the Process Layer, the descriptions defined on the Information Layer, and descriptions defined on the System Layer can be applied. The Information Layer describes the message types for registration and query. An IDS Metadata Broker may provide additional services that in term must be described by using terms from the IDS Information Model in the respective Metadata Broker's Self-Description document.
+The IDS Metadata Broker consists of an IDS Connector (see Section [3.5.2](./3_5_2_IDS_Connector.md#ids-connector)), an endpoint for the registration, publication, maintenance, and query of Self-Descriptions.
```
Comment (sebbader-sap) [inline documentation/3_Layers_of_the_Reference_Architecture_Model/3_5_System_Layer/3_5_4_Metadata_Broker.md:3]: I would rather say that "The IDS Metadata Broker is an IDS Connector (see Section [3.5.2](./3_5_2_IDS_Connector.md#ids-connector)), which contains an endpoint for the registration ..."

File: documentation/3_Layers_of_the_Reference_Architecture_Model/3_5_System_Layer/3_5_4_Metadata_Broker.md [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
+The IDS Metadata Broker consists of an IDS Connector (see Section [3.5.2](./3_5_2_IDS_Connector.md#ids-connector)), an endpoint for the registration, publication, maintenance, and query of Self-Descriptions. 
+
+A Self-Description encapsulates information about the component hosted within an IDS Connector. This Self-Description contains information about the offered interfaces, the owner of the component and the metadata of the data offered by the component. A Self-Description is provided by the owner of a component. The Self-Description in total can be seen as metadata.<br>
```
Comment (sebbader-sap) [inline documentation/3_Layers_of_the_Reference_Architecture_Model/3_5_System_Layer/3_5_4_Metadata_Broker.md:5]: A Self-Description encapsulates information about the IDS Connector itself and its capabilities and characteristics. ... A Self-Description is provided by the operator of the Connector. ...

File: documentation/3_Layers_of_the_Reference_Architecture_Model/3_5_System_Layer/3_5_4_Metadata_Broker.md [hunk 3]
```diff
@@ -1,12 +1,17 @@
...
+An IDS Connector providing a service or data can send its Self-Description to a IDS Metadata Broker so that every participant is able to find it within the dataspace. The IDS Metadata Broker can be understood as a phone book. Within a dataspace, there can be multiple IDS Metadata Brokers allowing to decentralize the IDS Metadata Broker.
```
Comment (sebbader-sap) [inline documentation/3_Layers_of_the_Reference_Architecture_Model/3_5_System_Layer/3_5_4_Metadata_Broker.md:6]: ... Within a dataspace, there can be multiple IDS Metadata Brokers allowing to distribute the IDS Metadata Broker functionality. It is up to the dataspace authority to decide if there is a leading IDS Metadata Broker or if the different instances operate independently.

File: documentation/3_Layers_of_the_Reference_Architecture_Model/3_5_System_Layer/3_5_4_Metadata_Broker.md [hunk 4]
```diff
@@ -1,12 +1,17 @@
...
+
+A participant can interact with an IDS Metadata Broker by using the processes defined on the Process Layer, the descriptions defined on the Information Layer, and descriptions defined on the System Layer. The Information Layer describes the message types for registration and query. An IDS Metadata Broker may provide additional services that in term must be described by using terms from the IDS Information Model in the respective Metadata Broker's Self-Description document.
```
Comment (sebbader-sap) [inline documentation/3_Layers_of_the_Reference_Architecture_Model/3_5_System_Layer/3_5_4_Metadata_Broker.md:8]: ... The Information Layer describes the message types for registration and query as well as their content. ...

**Review by sebbader — APPROVED** [review]

File: documentation/3_Layers_of_the_Reference_Architecture_Model/3_5_System_Layer/3_5_4_Metadata_Broker.md [hunk 5]
```diff
@@ -1,12 +1,17 @@
...
+The IDS Metadata Broker is an IDS Connector (see Section [3.5.2](./3_5_2_IDS_Connector.md#ids-connector)), which contains an endpoint for the registration, publication, maintenance, and query of Self-Descriptions. 
+
+A Self-Description encapsulates information about IDS Connector itself and its capabilities and characteristics. This Self-Description contains information about the offered interfaces, the owner of the component and the metadata of the data offered by the component. A Self-Description is provided by the operator of the Connecotr. The Self-Description in total can be seen as metadata.<br>
```
Comment (ssteinbuss) [inline documentation/3_Layers_of_the_Reference_Architecture_Model/3_5_System_Layer/3_5_4_Metadata_Broker.md:5]: Typo in Connector, please avoid mixing HTML tags with the text. use an empty line instead

**Review by ssteinbuss — CHANGES_REQUESTED** [review]
Thanks a lot for contributing here. Contentwise great input, please check again for typos and remove HTML tags mixed with the text.

**Review by ssteinbuss — APPROVED** [review]

**Pull request merged**